eudemo/eudemo/ivc/divertor_silhouette.py

```diff
@@ -199,7 +199,8 @@ def make_dome(
         """
         # Get the flux surface that crosses the through the start point
         # We can use this surface to guide the shape of the dome
-        psi_start = self.equilibrium.psi(*start)
+
+        psi_start = self.equilibrium.psi(*(start if start[1] < end[1] else end))
```

Comment:
Can you put a comment on this, please? I saw something about why somewhere, but if it's not tied to this line we're going to forget!